modules/simple_bucket/metadata.yaml

```diff
@@ -85,12 +85,12 @@ spec:
               source: github.com/GoogleCloudPlatform/terraform-google-cloud-run//modules/v2
               version: ~> 0.13
             spec:
-              outputExpr: '{"member": service_account_id.member, "role": "roles/storage.objectAdmin"}'
+              outputExpr: "{\"member\": service_account_id.member, \"role\": \"roles/storage.objectAdmin\"}"
```

Comment:
Was this causing any issue? We need this string without any `\` character in ADC..

Reply:
Weird, not sure why CFT populates these fields differently than before. Didn't know ADC does not accept escapes in strings.

There are a couple of `\n`s in `internal_encryption_config` as well - do we need to delete them? If so how do we handle new lines?

Reply:
Cool! Not needed. This is just for `outputExpr` of connection metadata